client/package.json

```diff
@@ -115,27 +110,26 @@
   },
   "scripts": {
     "vscode:prepublish": "tsc -p ./",
-    "compile": "tsc -watch -p ./",
+    "compile": "tsc -p ./",
+    "compile-watch": "tsc -watch -p ./",
     "postinstall": "node ./node_modules/vscode/bin/install",
     "test": "node ./node_modules/vscode/bin/test"
   },
+
   "devDependencies": {
     "typescript": "^2.0.3",
     "vscode": "^1.0.0",
-    "vscode-languageclient": "^3.1.0",
     "vsce": "^1.18.0",
-    "mocha": "^2.3.3",
+
+    "@types/node": "^6.0.40",
```

Comment:
Not sure changing order here helps, confusing in retrospect which change fixed things